### DISCUSSION TITLE
Hyperpriors in MMM

### DISCUSSION THREAD

**Discussion by ldefour** [google/lightweight_mmm#36] [Q&A] [answered]
Hi @pabloduque0,

First of all, thanks for putting energy in creating this great library :)
I'm just starting with it, but I hope to contribute to it in a near future.

I went through `models.py` to understand how the geolevel model was actually defined and I wanted to ask a quick clarification question.

Is it correct to say that currently only the $\beta_{m,g}$ coefficients are having hyperpriors?
![image](https://user-images.githubusercontent.com/7029948/177737245-9a1b4895-d73c-4f08-9a99-145419b98883.png)

For instance, it seems to me that the intercepts are defined at the geo level but we do not assume that each geo intercept is sampled from a national intercept.
![image](https://user-images.githubusercontent.com/7029948/177737374-2f42ee33-d214-4000-a7d5-8a84a0576700.png)

Many thanks in advance for your answer and amazing work so far!

**Comment by pabloduque0** [comment, marked as answer]
Hey @ldefour ! its great you open this discussion, great question!

Yes, you are correct, the coefficients of the media parameters are the only ones set up with hyperpriors in that way.

Bear in mind that I will rename and reformat a couple things in that file in the next few weeks :)
